docker/subset-fastq-dataset/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.6.2
+
+LABEL maintainer="Mint Team <mintteam@broadinstitute.org>" \
+      software="python 3.6.2" \
+      description="python 3.6.2 with pysam, sctools, requests, and a basic science stack"
+
+RUN pip3 install \
+    Click==7.0 \
+    numpy==1.16.2 \
+    pysam==0.15.2 \
+    biopython==1.73
+
+## Make directory for tools and put them in the path
+RUN mkdir /tools
+ENV PATH=/tools/:${PATH}
+
+## Note that this string is specific this linux base
+ENV CLOUD_SDK_REPO="cloud-sdk-jessie"
```

Comment:
Does this need to be an environment variable or can it just be inlined in the below `RUN` usage?

Reply:
It's easier to change in the future if it is standalone, if the base ubuntu image is ever changed this will require updating as well.